diesel/src/pg/types/geometric.rs

```diff
@@ -0,0 +1,62 @@
+//! Support for Geometric types under PostgreSQL.
+
+use byteorder::{NetworkEndian, ReadBytesExt, WriteBytesExt};
+use std::error::Error;
+use std::io::prelude::*;
+
+use deserialize::{self, FromSql};
+use expression::AsExpression;
+use expression::bound::Bound;
+use pg::Pg;
+use serialize::{self, Output, ToSql, IsNull};
+use sql_types::{Point};
+
+/// Point is represented in Postgres as a tuple of 64 bit floating point values (x, y).  This
+/// struct is a dumb wrapper type, meant only to indicate the tuple's meaning.
+#[derive(Debug, Clone, PartialEq, Copy, FromSqlRow)]
+pub struct PgPoint(pub f64, pub f64);
```

Comment:
Should this be just a raw tuple i.e. `(f64, f64)`?

Reply:
Nope.